keras/engine/training.py

```diff
@@ -945,9 +946,25 @@ def handle_metrics(metrics, weights=None):
         trainable_weights = self.trainable_weights
         self._collected_trainable_weights = trainable_weights
 
+    def check_trainable_weights_consistency(self):
```

Comment:
Please make this method private (underscore) and make the docstring style-compliant.